### PR TITLE
[new release] catt (3 packages) (1.0)

### PR DESCRIPTION
**Pull request by thibautbenjamin**
An infinity-categorical coherence typechecker

- Project page: <a href="https://github.com/thibautbenjamin/catt">https://github.com/thibautbenjamin/catt</a>

## Coq catt plugin
- Working export of catt term into coq

## Catt
- Computation of 1-naturality
- Computation of functorialisation
- Computation of inverses and cancellation witnesses
- Computation of opposites
- Builtin identities and compositions
- Computation of suspension and implicit suspension
- Inference of implicit variables
- Basic type checker

### CODE REVIEW

File: packages/catt-web/catt-web.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+maintainer: ["Thibaut Benjamin" "Chiara Sarti"]
+authors: ["Thibaut Benjamin" "Chiara Sarti"]
+license: "MIT"
+tags: ["higher-categories" "dependent-type-theory"]
+homepage: "https://github.com/thibautbenjamin/catt"
+bug-reports: "https://github.com/thibautbenjamin/catt/issues"
+depends: [
+  "dune" {>= "3.16"}
+  "odoc" {with-doc}
+]
```
Comment (mseri) [inline packages/catt-web/catt-web.1.0/opam:13]:
```suggestion
  "catt" {= version}
]
```

File: packages/catt/catt.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+authors: ["Thibaut Benjamin" "Chiara Sarti"]
+license: "MIT"
+tags: ["higher-categories" "dependent-type-theory"]
+homepage: "https://github.com/thibautbenjamin/catt"
+bug-reports: "https://github.com/thibautbenjamin/catt/issues"
+depends: [
+  "ocaml" {>= "4.14" & < "5.0"}
+  "dune" {>= "3.16" & >= "3.10"}
+  "menhir" {>= "20240715"}
+  "base" {>= "0.17.0"}
```
Comment (mseri) [inline packages/catt/catt.1.0/opam:14]:
```suggestion
  "landmarks"
  "base" {>= "0.17.0"}
```

**Comment by thibautbenjamin** [comment]
Hi, apologies for the delay for the reply, and thank you for the suggestion. This is the first time I am publishing a package, so forgive me if some things are not optimal. Here is my situation:
- I have used dune-release to create this merge request. As it happens, dune-release added the opam file for the  `catt-web` package. This package does not install anything, it just generates javascript code, and we are using internally, but I do not believe it should be on opam.
- We have some dependency problem with the landmarks library, that we use for profiling. I do not want to add landmarks as a mandatory dependency, as it is not necessary for the code to run smoothly. I thought I found a way to have landmarks be an optional dependency with dune, but evidently I failed in doing so.
- Since I submitted this pull request, I have been informed that the parsing we are doing fails on windows generated files. We now have a fix.

I will try to solve all those three problems and update this merge request accordingly

**Comment by mseri** [comment]
Thanks!

**Comment by thibautbenjamin** [comment]
I made the changes that I wanted. Hopefully it passes the CI. Please let me know if there is anything else I should do

**Comment by thibautbenjamin** [comment]
I am seeing there is still a compilation issue. One of the issues might be that one of the package is a coq plugin. I just find out that there was a separate repository for these. Should I remove the coq-plugin from this MR to only have the base package, and open an MR on the dedicated coq-plugins repo?

**Comment by mseri** [comment]
No, it looks all right. The failure is due to a strict lint that is there for our own sake (to avoid certain package names collisions). Would it be possible to have one of the maintainers email in the opam file?

**Comment by thibautbenjamin** [comment]
I just added our emails! Let me know if there is anything else I should do

**Comment by mseri** [comment]
Thanks a lot!

**Pull request merged**